articles/storage-xplat-guide-full.md

```diff
@@ -111,79 +112,91 @@ While most of the storage commands will work without an Azure subscription, we r
 ### How to create a new Azure storage account
 To use Azure storage, you will need a storage account. You can create a new Azure storage account after you have configured your computer to connect to your subscription. 
 
-        azure storage account create
+        azure storage account create <account_name>
 
 > [AZURE.IMPORTANT] The name of your storage account must be unique within Azure and must be lowercase. For naming conventions and restrictions, see [About Azure Storage Accounts](storage-create-storage-account.md) and [Naming and Referencing Containers, Blobs, and Metadata](http://msdn.microsoft.com/library/azure/dd135715.aspx).
```

Comment:
The link: http://msdn.microsoft.com/library/azure/dd135715.aspx is not about naming of storage accounts. 
 The statement of storage accounts name I can find is: "A name for the storage account that is unique within Azure. Storage account names must be between 3 and 24 characters in length and use numbers and lower-case letters only." (https://msdn.microsoft.com/en-us/library/azure/hh264518.aspx)